src/views/language/LanguageTable.tsx

```diff
@@ -112,6 +159,112 @@ const LanguageTable: React.FC = () => {
           sortParam: SortBy.PopulationOfDescendents,
           columnGroup: 'Population',
         },
+        {
+          key: 'Vitality (Meta)',
+          label: 'Vitality (Meta)',
+          render: (lang) => {
+            const { score, explanation } = computeVitalityMetascore(lang);
+            const b = scoreBucket(score);
+            return (
+              <Hoverable hoverContent={explanation}>
+                <div
+                  style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    gap: '0.35em',
+                    color: bucketColor(b),
+                  }}
+                >
+                  {bucketIcon(b)}
+                  <span style={{ fontVariantNumeric: 'tabular-nums' }}>{scoreLabel(score)}</span>
+                </div>
+              </Hoverable>
+            );
+          },
+          isNumeric: true,
+          columnGroup: 'Vitality',
+        },
+        {
+          key: 'Vitality (ISO)',
+          label: 'Vitality (ISO)',
```

Comment:
probably better to call this Vitality: ISO
```suggestion
          key: 'Vitality: ISO',
```